lte/gateway/deploy/roles/magma/files/envoy.yaml

```diff
@@ -31,10 +36,15 @@ node:
 static_resources:
   clusters:
   - connect_timeout: 1s
-    hosts:
-    - socket_address:
-        address: 10.5.0.2
-        port_value: 18000
+    load_assignment:
+      cluster_name: xds_cluster
+      endpoints:
+      - lb_endpoints:
```

Comment:
I think this line and all dependent lines need to go on more step right to make yamllint happy.